### ISSUE TITLE
Contacts not displayed if only one letter is typed

### ISSUE THREAD

**Issue by ohshaban** [lolobosse/ContactsAutoCompleteTextView#11]
I realized that no contacts are displayed if only one letter is typed, there has to be a minimum of two letters for any search results to appear. Is there a way to fix this?

**Comment by lolobosse** [comment]
Yes, I guess it is a bug because [here](https://github.com/lolobosse/ContactsAutoCompleteTextView/blob/master/contactsautocompletetextview/src/main/java/com/meyerlaurent/cactv/AutoCompleteContactTextView.java#L276) it only check for 0 and not 1.

Need to investigate further,

I have not that much time yet but as soon as we release, I'll have a deep look and release 0.2
